### PR TITLE
Disable Ubuntu 20.04 for Uyuni, and some minor fixes

### PR DESCRIPTION
**Pull request by juliogonzalez**
**WARNING**: Do not merge https://github.com/uyuni-project/uyuni-docs/pull/243 until this PR (or something to disable/remove Ubuntu 20.04 on Uyuni) is merged!!!

I generated both docs, and looked good, but please have a review as well.

As you see, I commented where I could, and added `if`s where I couldn't.

I really think this situation is better than rolling back, specially because there are some places where you can't just remove lines. Enabling this when we have Ubuntu 20.04 on Uyuni will be really quick.

### CODE REVIEW

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 1]
```diff
@@ -165,13 +171,11 @@ endif::[]
 ifeval::[{suma-content} == true]
 .Procedure: Alternative Method for Adding {ubuntu} Channels and Repositories
 . At the command prompt on the {productname} Server, as root, install the [systemitem]``spacewalk-utils`` package:
-+
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-ubuntu.adoc:168]: I removed them, at the generated HTML doc the `+` was shown, and I guess that is not wanted.

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 2]
```diff
@@ -247,18 +247,18 @@ ubuntu-1804-amd64-universe-updates-uyuni \
...
-spacewalk-common-channels \
-ubuntu-2004-pool-amd64-uyuni \
-ubuntu-2004-amd64-main-uyuni \
-ubuntu-2004-amd64-main-updates-uyuni \
-ubuntu-2004-amd64-main-security-uyuni \
-ubuntu-2004-amd64-universe-uyuni \
-ubuntu-2004-amd64-universe-updates-uyuni \
-ubuntu-2004-amd64-uyuni-client
-----
+// Uncomment the following lines we add the Ubuntu 20.04 there (Leap 15.2 server is require first)
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/clients-ubuntu.adoc:250]: Commented as agreed.

File: modules/client-configuration/pages/supported-features-ubuntu.adoc [hunk 3]
```diff
@@ -28,7 +30,7 @@ The icons in this table indicate:
 | Operating system packages | icon:times[role="danger"] | icon:times[role="danger"] | icon:times[role="danger"]
 | Registration | ``Salt`` | ``Salt`` | ``Salt``
 | Install packages | ``Salt`` | ``Salt`` | ``Salt``
-| Apply patches | icon:question[role="gray"] | icon:question[role="gray"] | icon:question[role="gray"]
```
Comment (juliogonzalez) [inline modules/client-configuration/pages/supported-features-ubuntu.adoc:31]: Not sure what the character before `Apply` was, but it wasn't a whitespace:

image

File: modules/client-configuration/pages/clients-ubuntu.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/client-configuration/pages/clients-ubuntu.adoc:168]: Removing them will mean the code block pops out of the step block, though, and then renumbering will start again. All that said, they shouldn't be showing either :woman_shrugging:

Comment (juliogonzalez) [reply modules/client-configuration/pages/clients-ubuntu.adoc:168]: Let's merge, and maybe you can have a look on your branch :-)

**Pull request merged**